fetch/programmer-humour/script.js

```diff
@@ -0,0 +1,28 @@
+
+let myData;
```

Comment:
Could you improve the naming of this variable so that it reflects the type of data you receive?